components/widgets/menu/menu_modals/submenu_modal/submenu_modal.jsx

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import {Modal} from 'react-bootstrap';
+
+import MenuWrapper from '../../menu_wrapper';
+import Menu from '../../menu';
+
+export default class SubMenuModal extends React.PureComponent {
+    static propTypes = {
+        elements: PropTypes.array,
+        action: PropTypes.func,
+        onHide: PropTypes.func.isRequired,
+    }
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            show: true,
+        };
+    }
+
+    onHide = () => {
+        this.setState({show: false});
+    }
+
+    render() {
+        return (
+            <Modal
+                dialogClassName='a11y__modal mobile-sub-menu'
+                show={this.state.show}
+                onHide={this.onHide}
+                onExited={this.props.onHide}
+                enforceFocus={false}
+                id='submenuModal'
+                role='dialog'
+                aria-labelledby='submenuModalLabel'
```

Comment:
Where is the submenuModalLabel referenced here?

Reply:
There isn't currently, so I removed this.